data/json/items/comestibles/meat_dishes.json

```
@@ -723,6 +723,31 @@
     "vitamins": [ [ "vitC", 3 ], [ "calcium", 5 ], [ "iron", 21 ] ],
     "fun": 3
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "pemmicant",
```

```suggestion
    "id": "pemmican_meat",
```
Having clear ids is also nice for future contributors